argo_qa/schema.xml

```diff
@@ -11,55 +11,31 @@
 
     <!-- NOTE:  not all possible Solr field types are represented in the dynamic fields -->
 
-    <!-- text (_t...) -->
-    <dynamicField name="*_ti"    type="text" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tim"   type="text" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_ts"    type="text" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_tsm"   type="text" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_tsi"   type="text" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tsim"  type="text" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_tiv"   type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_timv"  type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsiv"  type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsimv" type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
-    <!-- English text (_te...) -->
-    <dynamicField name="*_tei"    type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_teim"   type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_tes"    type="text_en" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_tesm"   type="text_en" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_tesi"   type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tesim"  type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_teiv"   type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_teimv"  type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesiv"  type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesimv" type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
     <!-- string (_s...) -->
-    <dynamicField name="*_si"    type="string"    stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_sim"   type="string"    stored="true" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_si"    type="string"    stored="false" indexed="true"  multiValued="false"/>
```

Comment:
some years ago Chris Beer decided that all argo field types should be stored.  I think it was because access does solr index updates with partial documents.  But we do not.   

If we were going to keep everything stored, these definitions are duplicates of, e.g. _ssi